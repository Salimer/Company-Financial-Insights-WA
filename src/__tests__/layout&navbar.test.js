import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Layout from '../components/layout';
import Navbar from '../components/navbar';

describe('Layout', () => {
  test('renders correctly', () => {
    const { container } = render(
      <BrowserRouter>
        <Layout />
      </BrowserRouter>
    );
    expect(container).toMatchSnapshot();
  });
});

describe('Navbar', () => {
  test('renders correctly', () => {
    const { container } = render(
      <BrowserRouter>
        <Navbar />
      </BrowserRouter>
    );
    expect(container).toMatchSnapshot();
  });
});
