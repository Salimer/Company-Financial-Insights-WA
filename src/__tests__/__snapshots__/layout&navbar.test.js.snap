// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
<div>
  <header
    class="sc-bgqQcB jDvmPV shadow-md w-full top-0 left-0 bg-navbarBg border border-white"
  >
    <ul
      class="flex items-center justify-between py-4 px-10 text-white"
    >
      <a
        aria-current="page"
        class="active"
        href="/"
      >
        <ion-icon
          name="arrow-back-outline"
        />
      </a>
      <li
        class="px-4 text-center"
      >
        Income Statement viewer
      </li>
      <li
        class="flex gap-1 md:gap-6"
      >
        <span>
          <ion-icon
            name="mic-outline"
          />
        </span>
        <span>
          <ion-icon
            name="settings-outline"
          />
        </span>
      </li>
    </ul>
  </header>
</div>
`;

exports[`Navbar renders correctly 1`] = `
<div>
  <header
    class="sc-bgqQcB jDvmPV shadow-md w-full top-0 left-0 bg-navbarBg border border-white"
  >
    <ul
      class="flex items-center justify-between py-4 px-10 text-white"
    >
      <a
        aria-current="page"
        class="active"
        href="/"
      >
        <ion-icon
          name="arrow-back-outline"
        />
      </a>
      <li
        class="px-4 text-center"
      >
        Income Statement viewer
      </li>
      <li
        class="flex gap-1 md:gap-6"
      >
        <span>
          <ion-icon
            name="mic-outline"
          />
        </span>
        <span>
          <ion-icon
            name="settings-outline"
          />
        </span>
      </li>
    </ul>
  </header>
</div>
`;
